index.html

```diff
@@ -1,34 +1,56 @@
 <!doctype html>
 
 <html lang="en">
+
 <head>
-  <meta charset="utf-8">
+    <meta charset="utf-8">
 
-  <title>Sprint Challenge - Home</title>
+    <title>Sprint Challenge - Home</title>
 
-  <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
+    <link rel="stylesheet" href="css/index.css">
 
 </head>
 
 <body>
     <div class="container">
-   
+
+        <header class="header">
+            <nav>
+                <div class="logo">
+                    <img class="logo" src="img/lambda-black.png" alt="Lambda Logo">
+                </div>
+                <ul>
+                    <li>Home</li>
```

Comment:
Better and cleaner to use anchor **</a>** tags directly inside the nav. You just have to specify **text-decoration: none and set the font colors to match**